src/Generator/Traits/ChangelogTemplate.php

```diff
@@ -0,0 +1,174 @@
+<?php
+namespace Mill\Generator\Traits;
+
+use StringTemplate\Engine;
+
+trait ChangelogTemplate
+{
+    /**
+     * Changelog template output format. Acceptable values: "json", "html", or "markdown"
```

Comment:
feels like those three vals should be `const` and referenced instead of compared to strings throughout